decimal.md

```diff
@@ -27,11 +27,9 @@ When inserting a decimal value:
 
 ## Format
 
-When inserting a decimal value, format it as `DECIMAL '1.2345'`. This casts the value as a string, which preserves the number's exact precision.
+When inserting into a `DECIMAL` column, format the value as `1.2345`. CockroachDB considers the type of the column when handling literals, so non-decimal numbers are also accepted.
```

Comment:
I dont think I would say `non-decimal numbers are also accepted`. `1.2345` isnt a decimal, or a float, or any data type. It is an untyped literal that is being given its type based on how it is used.